exercises/concept/triple-mary/.docs/introduction.md.tpl

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+%{concept:namespaces}
```

Comment:
Template strings likely should have an end of line for the last line.  Otherwise, this may present problems on joining data (since "line 3" is not yet a line, as there is no end of line.

This can present "surprises" in practice.